docs/docs/guides/automate/sensors/testing-sensors.md

```diff
@@ -67,7 +73,7 @@ This sensor uses the `context` argument. To invoke it, we need to provide one:
 
 For sensors which utilize [resources](/guides/build/external-resources/), you can provide the necessary resources when invoking the sensor function.
 
-Below is a test for the `process_new_users_sensor` that we defined in "[Using resources in sensors](using-resources-in-sensors)", which uses the `users_api` resource.
+Below is a test for the `process_new_users_sensor` that we defined in "[Using resources in sensors](/guides/automate/sensors/using-resources-in-sensors)", which uses the `users_api` resource.
 
 {/* TODO add dedent=4 prop to CodeExample below when implemented */}
```

Comment:
I have an open PR that might result in merge conflicts here, just a heads up.

https://github.com/dagster-io/dagster/pull/28735

Reply:
Gotcha, I just approved yours, so feel free to merge and I can fix any conflicts in my branch, then merge mine.

Reply:
Thanks, done!